driver/src/metrics/stablex_metrics.rs

```diff
@@ -258,12 +258,19 @@ trait InitializeableMetric: 'static + Sized + AsRef<str> {
 }
 
 enum ProcessingStage {
+    /// The processing of a new batch has started
     Started,
+    /// The orderbook and account state has been fetched
     OrdersFetched,
+    /// A solution for the batch has been produced
     Solved,
+    /// The auction solution has been verified
     Verified,
+    /// The auction solution has been submitted
     Submitted,
-    Skipped,
+    /// Processing was successful, but solution was not submitted
+    /// (e.g. solution did not improve objective value)
+    SolutionNotSubmitted,
```

Comment:
I see we lost Skipped here and introduced `SolutionNotSubmitted`. Is this also used for the case when orderbook took too long to fetch?

Reply:
No, I removed the line where we log this event in case the orderbook too long. For those case we now get an orderbook_fetched as a terminal event.